docs/hub/spaces-advanced-usage.md

```diff
@@ -0,0 +1,94 @@
+---
```

Comment:
This page feels a bit as a new bag of very misc things. I feel 
*  Comparing different models
*  Duplicating a Space
* Creating a Space from a model
are a very special use case and might need to go in another page, most likely as a how to rather than in this kind of doc

Reply:
Good point! I think those three applications might even be better served as a blog post instead of documentation. It's not directly related to the Hub in the sense that it is a feature built into the Hub like Pull Requests and Discussions, or setting up a static Space as an Organization card. This seems like something someone might casually read when they're in "discovery" mode rather than referring to the docs because they need help with something.

Reply:
Yeah I see what you mean. I guess something I could do is ditch the "Advanced Usage" page altogether and move the contents to somewhere more relevant. I was thinking...

These get moved to each be a sub-page of Spaces Overview

> Using OpenCV in Spaces
Using Spaces for Organization cards
Additional ways to create Spaces (Duplicating a Space + Creating a Space from a model, combined)
Managing Spaces with Github Actions

And the **Comparing different models** doesn't feel like it fits in these docs, so I'd remove it.

What do you think?